demo/demo.py

```diff
@@ -68,14 +87,49 @@ def get_env(key, default=""):
 DEMO_HOST = get_env("LOOKER_DEMO_HOST", 'localhost')
 DEMO_PORT = int(get_env("LOOKER_DEMO_PORT", '8080'))
 
+# 
+# Initialize Looker API SDK 
+# 
+
+looker_api_sdk = looker_sdk.init40(config_file = "./demo/looker.ini") 
+
+#
+# Function to create a signed URL using the Looker API SDK
+#
+
+def api_create_signed_url(embed_url, user, HOST, looker_api_sdk):
+  target_url =  'https://' + HOST + urllib.parse.unquote_plus(embed_url)
+  # the current front end sends a query with embed included, which we remove here
+  target_url = target_url.replace("embed/", "")
```

Comment:
Is this intentional? Don't we want the embed to be in the URL? Or does the create_sso_embed_url endpoint try to add it for us?

Reply:
AFAICT the create_sso_embed_url tries to add it for us, and the embed SDK also tries to add it, so its duplicated... its not elegant but removing it the handler resolved the duplication.

Reply:
LGTM